bundles/SyliusThemeBundle/your_first_theme.rst

```diff
@@ -0,0 +1,140 @@
+Your first theme
+================
+
+Themes location and definition
+------------------------------
+
+Private themes should be added to ``app/themes`` directory by default. Every theme should have a default confguration
+located in ``composer.json`` file. The only required parameter is ``name``, but it is worth to define other.
+
+.. code-block:: json
+
+    {
+      "name": "vendor/default-theme"
+    }
+
+When adding or removing theme, it's necessary to rebuild the container (same as adding new translation files in Symfony2) by clearing the cache (`app/console cache:clear`).
```

Comment:
@ahilles107 We need it to regenerate templates paths, translations and few more things. Supporting themes loaded at runtime is something we would love to have too, but not that easy to handle... We will look at that in future versions.